__tests__/Index.test.js

```diff
@@ -0,0 +1,9 @@
+import Index from '../app/js/openmrs-owa-platformui';
+
+describe('Index component', () => {
+  it('renders without breaking', () => {
+    expect(JSON.stringify(
+      Object.assign({}, Index, { _reactInternalInstance: 'censored' }),
+    )).toMatchSnapshot();
+  });
+});
```

Comment:
what are you trying to do here.

Reply:
Since the Index.js file does not actually export anything like a function to test but just renders the application to the DOM, I did not want to exclude it from the tests. So I JSON stringified it and did a snapshot test. To get more context, read more here:
https://wetainment.com/testing-indexjs/
https://stackoverflow.com/questions/43044696/test-a-create-react-app-index-js-file

Reply:
Since the Index.js file does not actually export anything like a function to test but just renders the application to the DOM, I did not want to exclude it from the tests. So I JSON stringified it and did a snapshot test. To get more context, read more here:
https://wetainment.com/testing-indexjs/
https://stackoverflow.com/questions/43044696/test-a-create-react-app-index-js-file